hw/formal/README.md

```diff
@@ -1,7 +1,4 @@
----
-title: "OpenTitan Assertions"
-unlisted: true
```

Comment:
This file is unlisted. How are we guaranteeing that? Maybe we should add a comment in the file?

Reply:
Yep, only files in the SUMMAR.md are in the table of contents. (Still technically accessible until mdbook ignore is sorted.)